### ISSUE TITLE
Accessibility: Datagrid does not announce selected row

### ISSUE THREAD

**Issue by 508it** [infor-design/enterprise#2031]
**Describe the bug**
The selected column in the datagrid is announced by AT as "row selected" instead of what user is selecting. It may be advisable to add the name of the item user is selecting to the select span name.

**To Reproduce in Landmark**
1. Go to https://hcm-tampm01-prd.inforcloudsuite.com/hcm/EmployeeSelfService/form/Employee%287004,700417%29.LRCFullEmployeeProfile?csk.showusingxi=true&csk.JobBoard=INTERNAL&csk.HROrganization=7004&menu=LRCEmployeeMenu.MyProfile
2. Click on any magnifying glass icon to launch a modal dialog
3. While using AT, select a row within the datagrid 
4. AT does not announce the selected row. 

**How to Fix for EP**
- This example has the same problem: http://master-enterprise.demo.design.infor.com/components/datagrid/example-mixed-selection.html
- Fix this so that selected row is announced with some setting like a primary id column, that is specified and that is read when the user tried to select a row.

**Expected behavior**
AT should announce the selected option and not just that a row is selected. 

**Additional context**
- component library example has the same problem: http://master-enterprise.demo.design.infor.com/components/datagrid/example-mixed-selection.html
- see also: https://jira.lawson.com/browse/LMCLIENT-24215

**Comment by tmcconechy** [comment]
Can i get a screen shot of this screen?

**Comment by 508it** [comment]
Below is an example of the datagrid in a modal dialog in Landmark:

<img width="431" alt="Screen Shot 2019-04-25 at 4 10 38 PM" src="https://user-images.githubusercontent.com/48891350/56765288-b1e5b780-6774-11e9-8a92-7fc7103adef0.png">

To recreate in the component library, follow the same steps from Landmark:
1. While using AT, select a row within the datagrid
2. AT does not announce the selected row

**Comment by tmcconechy** [comment]
So meaning..
- Tab into the grid to the checkbox color
- Listen for indication you can select
- Hit space to select

**Comment by 508it** [comment]
Those steps are correct.  The suggestion here is that we add a short metadata descriptor to append to the aria-label to give it more context when selecting.  For example, rather than just saying  "Select", we could say "Select 52106 Monitor"

**Comment by davidcarlsonberg** [comment]
I'm skeptical that this is a good idea to implement. What if the columns are different or reordered? Happy to be convinced otherwise, but it seems like a one-off example and for Accessibility one can arrow through the row to discover the different pieces of information.

**Comment by tmcconechy** [comment]
I was thinking that maybe..
- we add a settings.columId that is the id of the column that is the "important" one (i thought we had a grid "id" column setting but not seeing one on quick glance
- then this is added to the message in an audible span

**Comment by CindyMercadoReyes** [comment]
Tested this on Chrome Vox and it's working correctly. This issue is now resolved.
